src/modules/core/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -61,9 +62,16 @@ export function DropdownMenu<OptionData = undefined>({
                 render(item)
               ) : (
                 <>
-                  {item.icon && (
-                    <Icon name={item.icon} width={20} height={20} />
-                  )}
+                  {item.icon &&
+                    (typeof item.icon === 'string' ? (
```

Comment:
I suggest you to change it so we will have to always pass the `icon` prop as `ReactNode`. it this case we won't need this validation. Another solution would be change the naming, for example make `icon` prop represent the iconName of the Icon and create different prop `startdornmen` to add items as `ReactNode`